templates/iiif-manifest-compose.ecs.yml

```diff
@@ -2,8 +2,11 @@ version: '3'
 services:
   iiif_manifest:
     environment:
-      IIIF_MANIFESTS_BASE_URL: 'https://$$host/manifests/'
+      HONEYBADGER_API_KEY_MANAGEMENT: ${HONEYBADGER_API_KEY_MANIFEST}
```

Comment:
Do we want a separate honeybadger api key for the manifest service?  We probably don't want to call it "management" inside the container either way.

Reply:
yep, that's just a typo. good catch